spec/factories/teams.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :team do
     sequence(:name) { |n| "Patriots#{n}" }
-    city "New England"
+    location "New England"
```

Comment:
Yup I was getting a failing test saying something like `undefined method city=` for one of my tests. I thought I got rid of all of the city attributes but looking at the test I realized it was using a factory, so I checked that and low and behold found a line that should have had `location` instead of `city`.

I'm begining to see how helpful tests are. They can catch errors you would have never seen  without a test.